scripts/emergency/close_farms.js

```
@@ -0,0 +1,210 @@
+            }
+
+            let farmsInPage = 0;
+            for (const farm of farms) {
+                allFarmIdentifiers.push(farm.identifier);
+                farmsInPage++;
+            }
+            console.log(`Fetched ${farms.length} farms in this page. Found ${farmsInPage} farms. Total so far: ${allFarmIdentifiers.length}`);
+
+            if (farms.length === limit) {
+                const lastFarm = farms[farms.length - 1];
+                if (lastFarm && lastFarm.farm_identifier) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)